pkg/probe/system_sdn_connector.go

```diff
@@ -0,0 +1,62 @@
+package probe
+
+import (
+	"log"
+
+	"github.com/bluecmd/fortigate_exporter/pkg/http"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+type SystemSdnConnectorResults struct {
```

Comment:
Change `Sdn` to `SDN` as per https://github.com/golang/go/wiki/CodeReviewComments#initialisms